### ISSUE TITLE
Compile error; should I clone specific stable tag?

### ISSUE THREAD

**Issue by emin63** [20tab/UnrealEnginePython#62]
First of all, thanks for this awesome project. I'm very excited to play with it. I am getting a compile error when trying to build (relevant Visual Studio error shown below).

I notice that this is very actively developed. Is there a particular stable version I should be using?

I tried commit 3646583b08cf3e6efcc318293758b332e0e2f31b and got the error below.

Thanks again,
-Emin

===============================
C:\Users\Emin\Documents\Unreal Projects\GPMProto\Plugins\UnrealEnginePython\Source\UnrealEnginePython\Public\PyCommandlet.cpp(31): error C2440: 'initializing': cannot convert from 'initializer list' to 'TArray<FString,FDefaultAllocator>'
2>  C:\Users\Emin\Documents\Unreal Projects\GPMProto\Plugins\UnrealEnginePython\Source\UnrealEnginePython\Public\PyCommandlet.cpp(31): note: No constructor could take the source type, or constructor overload resolution was ambiguous
2>ERROR : UBT error : Failed to produce item: C:\Users\Emin\Documents\Unreal Projects\GPMProto\Plugins\UnrealEnginePython\Binaries\Win64\UE4Editor-UnrealEnginePython.dll
2>  Total build time: 42.19 seconds
2>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V140\Microsoft.MakeFile.Targets(37,5): error MSB3075: The command ""C:\Program Files\Epic Games\4.12\Engine\Build\BatchFiles\Build.bat" GPMProtoEditor Win64 Development "C:\Users\Emin\Documents\Unreal Projects\GPMProto\GPMProto.uproject" -waitmutex" exited with code 5. Please verify that you have sufficient rights to run this command.
========== Build: 1 succeeded, 1 failed, 0 up-to-date, 0 skipped ==========

**Comment by rdeioris** [comment]
can you try building it with unreal 4.13 instead of 4.12 ?

**Comment by emin63** [comment]
Upgrading to 4.13 fixed the compile issue. Things compile and link correctly now. Thanks!

A few more minor comments follow in case they help you or other users of this awesome package.

  1. When I first got things to compile & link and started my UE4 project I got an error about the ue_site python module not being found.
  2. I created ue_site.py inside Content/Scripts/ and this silenced the error but things were not quite working correctly. That is, I could not get the funnygameclasses example in your README to work. I closed UE4, added a few debug statements to your C++ code, did "Rebuild Solution", and restarted my UE4 project and everything worked. Probably I needed to create the ue_site.py python module inside Content/Scripts before starting UE4.
  3. I was still getting the warning (`LogDirectoryWatcher:Warning: Failed to begin reading directory changes for ../../../../../../Users/Emin/Documents/Unreal Projects/GPMProto/Plugins/UnrealEnginePython/Content/. Error: The system cannot find the file specified. (0x00000002)
`) which I was able to fix by adding an empty Content directory inside /Plugins/UnrealEnginePython. Am I supposed to put something in /Plugins/UnrealEnginePython/Content?

  4. I got all this working using Anaconda with Python 3.5.1 as I use that for other things on my system. Nice that everything works fine with that too.

To summarize: awesome project. I can't tell you how excited I am to be able to use so much UE4 functionality via Python. You may want to update the README at https://github.com/20tab/UnrealEnginePython to mention to use 4.13 not 4.12.5 and to create the ue_site.py and empty Content directory before trying to use the plugin.

Thanks,
-Emin

**Comment by rdeioris** [comment]
Hi, the latest commit reintroduce 4.12 support
